dietstory-routing/registration/views.py

```diff
@@ -670,5 +674,100 @@ def post(self, request, *args, **kwargs):
         
         logger.warn("[RV-10] Invalid input parameters")
         return JsonResponse({'message': "Invalid input parameters"}, status=status.HTTP_400_BAD_REQUEST)
+    
+    class DisconnectFromServerView(views.APIView):
 
+        permission_classes = (permissions.IsAuthenticated,)
+
+class DisconnectView(views.APIView):
+    permission_classes = (permissions.AllowAny,)
 
+    """
+    summary: Disconnect character
+    description: Disconnect character based on their characters name
+                 and what world they are in.
+    parameters:
+        - name: character_id
```

Comment:
`character_id` and `name of character` are not the same thing. Could we perhaps rename the parameter to avoid this confusion?

Reply:
Yea sounds good . Renamed to character_name.